solara/components/markdown.py

```diff
@@ -76,6 +76,12 @@ def _markdown_template(
     html,
     style="",
 ):
+    cdn = r"null"
```

Comment:
```suggestion
    cdn = None
```

Reply:
Let me know if you agree with my solution - I changed `cdn` to `None` by default, and instead filter it out later:
```python
        this.cdn = """
        + (rf"'{cdn}'" if cdn is not None else r"null")
        + r""";
```

Reply:
yeah, this makes more sense 👍